modules/claims_api/app/services/claims_api/slack/failed_submissions_messenger.rb

```diff
@@ -111,6 +118,16 @@ def datadog_timestamps
 
         [three_days_ago, current]
       end
+
+      # TID value is more a string blob of various data that follows this format (including quotes):
+      # 'Form526Submission_3443656, user_uuid: [filtered], service_provider: lighthouse'
+      # The KV-looking stuff isn't useful in a DD link, so extract the "tag" at the beginning of the string
+      def extract_tag_from_whitelist(id)
+        return nil if TID_SUBSTRING_WHITELIST.none? { |s| id.upcase.include? s }
+
```

Comment:
Fails on nil
option: `        return nil if TID_SUBSTRING_WHITELIST.none? { |s| id&.upcase&.include? s }`

Reply:
That was trippy. The `let` was caching the final value of the array and would never actually test the other TID values I'd set up. Fixed [9f55869](https://github.com/department-of-veterans-affairs/vets-api/pull/21235/commits/9f55869eb92c5d800532c48eec5b7960d7859918)